test/pipelines-it-kubeflow.yml

```diff
@@ -38,6 +38,16 @@ jobs:
       source install.sh
     displayName: 'Install nni toolkit via source code'
 
+  - script: |
+      python3 -m pip install scikit-learn==0.20.0 --user
+      python3 -m pip install torch==0.4.1 --user
+      python3 -m pip install torchvision==0.2.1 --user
+      python3 -m pip install keras==2.1.6 --user
+      python3 -m pip install tensorflow-gpu==1.10.0 --user
+      sudo apt-get install swig -y
+      nnictl package install --name=SMAC
+    displayName: 'Install dependencies for integration tests'
```

Comment:
Should we consider to add mode into the displyaName message, like 'Install dependencies for integration tests in Kubeflow mode'

Reply:
Thanks, updated.